scripts/exglobal_cleanup.sh

```diff
@@ -0,0 +1,214 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+
+###############################################################
+# Clean up previous cycles; various depths
+# PRIOR CYCLE: Leave the prior cycle alone
+# shellcheck disable=SC2153
+GDATE=$(${NDATE} -"${assim_freq}" "${PDY}${cyc}")
+
+# PREVIOUS to the PRIOR CYCLE
+GDATE=$(${NDATE} -"${assim_freq}" "${GDATE}")
+gPDY="${GDATE:0:8}"
+gcyc="${GDATE:8:2}"
+
+# Remove the TMPDIR directory
+# TODO Only prepbufr is currently using this directory, and all jobs should be
+#   cleaning up after themselves anyway
+COMIN="${DATAROOT}/${GDATE}"
+[[ -d ${COMIN} ]] && rm -rf "${COMIN}"
+
+if [[ "${DELETE_COM_IN_ARCHIVE_JOB:-YES}" == NO ]] ; then
+    exit 0
+fi
```

Comment:
See other comment about whether this variable should be something like `DELETE_COM`.

I'm guessing this check is here because the cleanup job will always be in the mesh but users can turn off cleanup for whatever reason?

Reply:
Yeah. Since cleanup does very little with it off, in the future we probably should just turn off the job. But I'm leaving that outside the scope of this PR.